lib/wca_live/scoretaking/advancing.ex

```diff
@@ -210,6 +212,32 @@ defmodule WcaLive.Scoretaking.Advancing do
     end)
   end
 
+  defp clinched_qualifying_results(round) do
+    # Assume best possible attempts for empty results and see who of
+    # the currently entered results would still qualify.
```

Comment:
FTR we don't need to worry about results dropping out, because getting a better result than everybody is a "stronger assumption". Intuitively, when someone gets a better result than everybody they make 1 person not advance, but when they drop out they effectively make 0.75 person not advance.